samples/unit-tests/pointer/click/demo.js

```diff
@@ -1,3 +1,69 @@
+function test(type, tooltipEnabled) {
```

Comment:
This is polluting the global scope, which may come back and haunt us later. Wrap the `test` function and its calls (only the changes in this diff) in an iife.